test/LibraryManager.Test/LibrariesValidatorTest.cs

```diff
@@ -33,43 +34,77 @@ public void Setup()
         }
 
         [TestMethod]
-        public void DetectConlictsAsync_ConflictingFiles()
+        public async Task DetectConlictsAsync_ConflictingFiles()
         {
+            string expectedErrorCode = "LIB016";
+            string expectedErrorMessage = "Conflicting file \"lib\\jquery.js\" found in more than one library: jquery@3.1.1, jquery@2.2.1";
             _manifest = Manifest.FromJson(_docConflictingLibraries, _dependencies);
 
-            var conflictDetector = new LibrariesValidator(_dependencies, _manifest.DefaultDestination, _manifest.DefaultProvider);
-
-            IEnumerable<FileConflict> conflicts = conflictDetector.GetFilesConflicts(_manifest.Libraries, CancellationToken.None);
+            IEnumerable<ILibraryOperationResult> conflicts = await LibrariesValidator.GetManifestErrorsAsync(_manifest, _dependencies, CancellationToken.None);
 
             Assert.AreEqual(1, conflicts.Count());
+            Assert.IsTrue(conflicts.First().Errors.Count() == 1);
+            Assert.AreEqual(conflicts.First().Errors.First().Code, expectedErrorCode);
+            Assert.AreEqual(conflicts.First().Errors.First().Message, expectedErrorMessage);
+        }
 
-            Assert.AreEqual("lib\\jquery.js", conflicts.First().File);
+        [TestMethod]
+        public async Task DetectConflictsAsync_SameLibraryDifferentFiles()
```

Comment:
>SameLibrary [](start = 47, length = 11)

Can you make this use different library names?  Since we will be changing the design to prevent multiple instances of the same library.